Mathlib/CategoryTheory/Sums/Basic.lean

```diff
@@ -147,8 +148,10 @@ end Swap
 
 end Sum
 
-variable {A : Type u₁} [Category.{v₁} A] {B : Type u₁} [Category.{v₁} B] {C : Type u₁}
-  [Category.{v₁} C] {D : Type u₁} [Category.{v₁} D]
+variable {A : Type u₁} [Category.{v₁} A] {A' : Type u₁} [Category.{v₁} A']
+  {B : Type u₂} [Category.{v₂} B]
+  {C : Type u₁} [Category.{v₁} C]
+  {D : Type u₂} [Category.{v₂} D]
```

Comment:
Is not it possible to have completely independent universes here?

Reply:
Same issue as in #22281: the category structure on the sum is only defined when both category live in the same universes, and it looks like it is a bit of a pain to generalize.

Reply:
It seems the category structure could be generalized to a category in `max v_1 v_2` at the cost of sprinkling ULifts everywhere. Is this really desirable?

Reply:
I'm not convinced the generalisation is worth it, particularly so because it makes the description of homs in the new category awkward: we can just ulift the two input categories from `v_1` and `v_2` to `max v_1 v_2` and take the sum there, now that they match.

Reply:
I believe the comment section of #22976 would be a more appropriate place to discuss this now that it is submitted.

Reply:
Oops, good point.